spec/system/header_search_spec.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+RSpec.describe "Header Search - Responsive Behavior", type: :system do
+  fab!(:user)
+  fab!(:topics) { Fabricate.times(20, :post).map(&:topic) }
```

Comment:
No need to do this, it's slow and we have a "cheat" way of doing this type of thing where you need to scroll a lot vertically:

https://github.com/discourse/discourse/blob/abee294616a064e4de40d8e8eb052ea571e2203c/spec/support/system_helpers.rb#L178-L187

https://github.com/discourse/discourse/blob/abee294616a064e4de40d8e8eb052ea571e2203c/spec/system/welcome_banner_spec.rb#L49-L49